cmd/proxy/Dockerfile

```diff
@@ -16,7 +16,8 @@ ARG VERSION="unset"
 
 RUN DATE="$(date -u +%Y-%m-%d-%H:%M:%S-%Z)" && GO111MODULE=on CGO_ENABLED=0 GOPROXY="https://proxy.golang.org" go build -ldflags "-X github.com/gomods/athens/pkg/build.version=$VERSION -X github.com/gomods/athens/pkg/build.buildDate=$DATE" -o /bin/athens-proxy ./cmd/proxy
 
-FROM alpine
+ARG ALPINE_VERSION=3.9.5
```

Comment:
This `ARG` here is unused, however I would just drop it and leave the image tag explicit as you do on the next line `FROM alpine:3.9.5` or `FROM alpine:3.9`. 

Unless you are building a factory for building docker images I would just be explicit :)

Reply:
Agreed, this should just be removed.

Reply:
@dsexton I'm gonna leave in the arg (after fixing https://github.com/gomods/athens/pull/1585#discussion_r395838521) on the off chance someone wants to do a custom build of the image. Some folks have wanted to do the same for the [go version](https://github.com/gomods/athens/blob/e89cf0feb2155f60ab2b0a808a75c1c84ccf8e34/cmd/proxy/Dockerfile#L8)